src/test/cypress/e2e/course/CourseManagement.cy.ts

```diff
@@ -41,7 +41,7 @@ const editedCourseData = {
 const allowGroupCustomization: boolean = Cypress.env('allowGroupCustomization');
 const dateFormat = 'MMM D, YYYY HH:mm';
 
-describe('Course management', () => {
+describe.skip('Course management', () => {
```

Comment:
```suggestion
describe.skip('Course management', () => {
```
You skip here the course management without any comment. It's also not mentioned in the PR. Accident?